sources/install/package_web.sh

```diff
@@ -723,6 +723,29 @@ function install_sqlmap() {
     add-to-list "sqlmap,https://github.com/sqlmapproject/sqlmap,Sqlmap is an open-source penetration testing tool that automates the process of detecting and exploiting SQL injection flaws"
 }
 
+function install_sqlisniper() {
+    colorecho "Installing SqliSniper"
+    git -C /opt/tools/ clone --depth 1 https://github.com/danialhalo/SqliSniper.git
+    python3 -m venv ./venv
+    ./venv/bin/python3 -m pip install -r requirements.txt
+    ln -s "/opt/tools/SqliSniper/sqlisniper.py" /opt/tools/bin/sqlisniper
```

Comment:
Create an alias to call the tool

`alias sqlisniper.py='/opt/tools/SqliSniper/venv/bin/python3 /opt/tools/SqliSniper/sqlisniper.py'`